Simplenote/BlockingActivityIndicator.swift

```diff
@@ -0,0 +1,61 @@
+import AppKit
+
+class BlockingActivityIndicator: NSView {
+    let activityIndicator: NSProgressIndicator
+
+    init(indicatorFrame: NSRect) {
+        self.activityIndicator = NSProgressIndicator(frame: indicatorFrame)
+        super.init(frame: .zero)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupView() {
+        wantsLayer = true
+        layer?.backgroundColor = NSColor.black.withAlphaComponent(0.3).cgColor
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Nitpicky: This one should probably live in `showIndicator`.

Usually tAMIC goes in the spot where the view is attached.